src/__tests__/components/NavBar/EditorDesignName.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import EditorDesignName from "../../../components/NavBar/EditorDesignName";
+
+describe("EditorDesignName", () => {
+  test("click the btn can edit the name", () => {
+    render(<EditorDesignName />);
+    const btnElement = screen.getByLabelText("Edit");
+    const spanElement = screen.getByText("Court Canva 1");
+    const inputElement = screen.getByDisplayValue("Court Canva 1");
+    fireEvent.click(btnElement);
+    expect(spanElement).toHaveAttribute("hidden");
+    expect(inputElement).not.toHaveAttribute("hidden");
+    fireEvent.change(inputElement, { target: { value: "new design name" } });
+    expect(inputElement).toHaveDisplayValue("new design name");
+    fireEvent.submit(inputElement);
+    expect(spanElement.hidden).toBeTruthy();
+    expect(inputElement.hidden).toBeFalsy();
+  });
+});
```

Comment:
Adding some empty newlines might make the code a little easier to read, just in my humble opinion

Reply:
I agree with Jin. Maybe adding more "it "should xxx"" and dividing this part into several independent pars would be more readable and maintainable.

Reply:
Thank you for the suggestion.  Update the file.